pkg/v3/plugin/ocr3.go

```diff
@@ -7,19 +7,27 @@ import (
 	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
 	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
 
+	ocr2keepers "github.com/smartcontractkit/ocr2keepers/pkg"
 	ocr2keepersv3 "github.com/smartcontractkit/ocr2keepers/pkg/v3"
 )
 
 type InstructionStore interface{}
 
 type SamplingStore interface{}
```

Comment:
remove

Reply:
remove the types? I think this is almost ready to happen with some other PRs coming through.